oap-server/server-starter/src/main/resources/otel-oc-rules/mysql.yaml

```diff
@@ -0,0 +1,84 @@
+# limitations under the License.
+
+# This will parse a textual representation of a duration. The formats
+# accepted are based on the ISO-8601 duration format {@code PnDTnHnMn.nS}
+# with days considered to be exactly 24 hours.
+# <p>
+# Examples:
+# <pre>
+#    "PT20.345S" -- parses as "20.345 seconds"
+#    "PT15M"     -- parses as "15 minutes" (where a minute is 60 seconds)
+#    "PT10H"     -- parses as "10 hours" (where an hour is 3600 seconds)
+#    "P2D"       -- parses as "2 days" (where a day is 24 hours or 86400 seconds)
+#    "P2DT3H4M"  -- parses as "2 days, 3 hours and 4 minutes"
+#    "P-6H3M"    -- parses as "-6 hours and +3 minutes"
+#    "-P6H3M"    -- parses as "-6 hours and -3 minutes"
+#    "-P-6H+3M"  -- parses as "+6 hours and -3 minutes"
+# </pre>
+filter: "{ tags -> tags.job_name == 'mysql-monitoring' }" # The OpenTelemetry job name
+expSuffix:  tag({tags -> tags.host_name = 'mysql::' + tags.host_name}).service(['host_name'] , Layer.MYSQL)
+metricPrefix: meter_mysql
+metricsRules:
+  # mysql configurations
+  - name: uptime
+    exp: mysql_global_status_uptime / 86400
+  - name: innodb_buffer_pool_size
+    exp: mysql_global_variables_innodb_buffer_pool_size
+  - name: max_connections
+    exp: mysql_global_variables_max_connections
+  - name: thread_cache_size
+    exp: mysql_global_variables_thread_cache_size
+
+  # database throughput
+  - name: commands_insert_rate
+    exp: mysql_global_status_commands_total.tagEqual('command','insert').rate('PT1M')
+  - name: commands_select_rate
+    exp: mysql_global_status_commands_total.tagEqual('command','select').rate('PT1M')
+  - name: commands_delete_rate
+    exp: mysql_global_status_commands_total.tagEqual('command','delete').rate('PT1M')
+  - name: commands_update_rate
+    exp: mysql_global_status_commands_total.tagEqual('command','update').rate('PT1M')
+  - name: qps
+    exp: mysql_global_status_queries.rate('PT1M')
+  - name: tps
+    exp: mysql_global_status_commands_total.tagMatch('command','rollback|commit').rate('PT1M')
+
+  # connections
+  ## threads
+  - name: threads_connected
+    exp: mysql_global_status_threads_connected
+  - name: threads_created
+    exp: mysql_global_status_threads_created
+  - name: threads_running
+    exp: mysql_global_status_threads_running
+  - name: threads_cached
+    exp: mysql_global_status_threads_cached
+  ## connect
+  - name: connects_aborted
+    exp: mysql_global_status_aborted_connects
+  - name: connects_available
+    exp: mysql_global_variables_max_connections.sum(['host_name']) - mysql_global_status_threads_connected.sum(['host_name'])
+  - name: connection_errors_max_connections
+    exp: mysql_global_status_connection_errors_total.tagMatch('error','max_connection')
+  - name: connection_errors_internal
+    exp: mysql_global_status_connection_errors_total.tagMatch('error','internal')
+
+  # slow queries
+  - name: slow_queries
+    exp: mysql_global_status_slow_queries
```

Comment:
You are showing a single value of this metric. In the current, you are actually showing the avg value of the duration. Such as avg value of the total number of slow queries in the selected 10 mins. This would make the description of this widget a little hard.

I think only `rate` has the real meaning.

Reply:
So I should delete this metric?

Reply:
I think it should be enough. Most counters are actually meaning a rate, because we want to see the trend.